CMakeLists.txt

```diff
@@ -613,10 +613,48 @@ add_test(NAME user_eg3
 # Define install
 # ---------------------------------------
 
-install(TARGETS ${qhull_TARGETS_INSTALL}
+install(TARGETS ${qhull_TARGETS_INSTALL} EXPORT QhullTargets
         RUNTIME DESTINATION ${BIN_INSTALL_DIR}
         LIBRARY DESTINATION ${LIB_INSTALL_DIR}
-        ARCHIVE DESTINATION ${LIB_INSTALL_DIR})
+        ARCHIVE DESTINATION ${LIB_INSTALL_DIR}
+        INCLUDES DESTINATION include)
+
+include(CMakePackageConfigHelpers)
+
+write_basic_package_version_file(
+    "${CMAKE_CURRENT_BINARY_DIR}/Qhull/QhullConfigVersion.cmake"
```

Comment:
This breaks the build on case-insensitive file systems (found on macOS) because `Qhull` directory name conflicts with `qhull` binary name:
```
[ 98%] Linking C executable qhull
ld: can't open output file for writing: qhull, errno=21 for architecture x86_64
clang-7: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [CMakeFiles/qhull.dir/build.make:85: qhull] Error 1
make[1]: *** [CMakeFiles/Makefile2:543: CMakeFiles/qhull.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
```
(`errno=21` is `EISDIR: Is a directory`)